driver-mapping/src/test/java/com/datastax/driver/mapping/MapperAsyncTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.mapping;
+
+import com.datastax.driver.core.CCMTestsSupport;
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.utils.MoreFutures.SuccessCallback;
+import com.datastax.driver.core.utils.UUIDs;
+import com.datastax.driver.mapping.annotations.Column;
+import com.datastax.driver.mapping.annotations.PartitionKey;
+import com.datastax.driver.mapping.annotations.Table;
+import com.google.common.base.Function;
+import com.google.common.base.Objects;
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.Uninterruptibles;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+
+import static com.datastax.driver.core.TestUtils.CREATE_KEYSPACE_SIMPLE_FORMAT;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.testng.Assert.fail;
+
+/**
+ * Basic tests for the mapping module.
+ */
+@SuppressWarnings("unused")
+public class MapperAsyncTest extends CCMTestsSupport {
```

Comment:
I get it now, the tests aren't being recorded because our second run with the 'isolated' profile is just running unit tests which blows away our reports.  I'll fix that, looks like it broke once I introduced that, my bad!